transformer_engine/pytorch/attention.py

```diff
@@ -6416,7 +6934,7 @@ class DotProductAttention(TransformerEngineBaseModule):
                can overlap two flash attention kernels.
     cp_comm_type : str
                   inter-gpu communication type for context parallelism.
-                  Can be "p2p" or "all_gather".
+                  Can be "p2p" or "all_gather" or "a2a".
```

Comment:
Could you add a bit more explanation about the three CP communication types please? It may help users quickly grasp what they are selecting.

Reply:
Added more detailed explanations to each of them. Thanks.